vue-components/src/components/ToggleButton.vue

```diff
@@ -0,0 +1,122 @@
+	</button>
+</template>
+
+<script lang="ts">
+import Vue, { VueConstructor } from 'vue';
+import { ToggleButtonGroupInjection } from '@/components/ToggleButtonGroup.vue';
+
+export default ( Vue as VueConstructor<Vue & ToggleButtonGroupInjection> ).extend( {
+	name: 'ToggleButton',
+	methods: {
+		onClick(): void {
+			if ( this.toggleListener !== null ) {
+				this.toggleListener( this.value );
+				return;
+			}
+			/**
+			 * only emitted when not use as part of a ToggleButtonGroup
+			 */
+			this.$emit( 'click' );
+		},
+	},
+	inject: {
+		groupValue: { default: null },
+		toggleListener: { default: null },
+	} as Record<keyof ToggleButtonGroupInjection, object>,
+	computed: {
+		buttonIsActive(): boolean {
+			if ( this.groupValue !== null ) {
+				return this.groupValue() === this.value;
+			}
+			return this.isActive;
+		},
+	},
+	props: {
+		/**
+		 * required when the ToggleButton is used as part of a ToggleButtonGroup.
+		 */
+		value: {
+			default: null,
+			type: String,
+		},
+		/**
+		 * required when the ToggleButton is used as a standalone component.
+		 * Ignored when used as part of a ToggleButtonGroup
+		 */
+		isActive: {
+			default: false,
+			type: Boolean,
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+.wikit-ToggleButton {
+	color: $wikit-ToggleButton-color;
+	font-family: $wikit-ToggleButton-font-family;
+	font-weight: $wikit-ToggleButton-font-weight;
+	font-size: $wikit-ToggleButton-font-size;
+	line-height: $wikit-ToggleButton-line-height;
+	border-width: $wikit-ToggleButton-border-width;
+	border-style: $wikit-ToggleButton-border-style;
+	border-radius: $wikit-ToggleButton-border-radius;
+	border-color: $wikit-ToggleButton-border-color;
+	background-color: $wikit-ToggleButton-background-color;
+	min-height: $dimension-min-height-medium;
+	padding-inline: $wikit-ToggleButton-medium-padding-horizontal;
+	padding-block: $wikit-ToggleButton-medium-padding-vertical;
+	transition-property: $wikit-ToggleButton-transition-property;
+	transition-duration: $wikit-ToggleButton-transition-duration;
+	transition-timing-function: $wikit-ToggleButton-transition-timing-function;
+
+	&:hover {
+		color: $wikit-ToggleButton-hover-color;
+		border-color: $wikit-ToggleButton-hover-border-color;
+		background-color: $wikit-ToggleButton-hover-background-color;
+	}
+
+	&:active {
+		color: $wikit-ToggleButton-active-color;
+		border-color: $wikit-ToggleButton-active-border-color;
+		background-color: $wikit-ToggleButton-active-background-color;
+	}
+
+	&:focus {
+		color: $wikit-ToggleButton-focus-color;
+		border-color: $wikit-ToggleButton-focus-border-color;
+		background-color: $wikit-ToggleButton-focus-background-color;
+		box-shadow: $wikit-ToggleButton-focus-box-shadow;
+	}
+
+	&:disabled {
+		color: $wikit-ToggleButton-disabled-color;
+		border-color: $wikit-ToggleButton-disabled-border-color;
+		background-color: $wikit-ToggleButton-disabled-background-color;
+	}
+
+	@media (max-width: $width-breakpoint-mobile) {
```

Comment:
Looking at storybook, I have the strong suspicion that this media-query is not working

Reply:
I double checked it before making the patch and it worked fine. Maybe you're checking against tablet breakpoint? The mobile breakpoint is pretty small.

Reply:
My bad, the media-query is indeed working fine!
It is that `box-sizing: border-box;` is missing. That is why the buttons are so strangely large ^^

The m-sized buttons are supposed to have a total height of 32px, not 44px :)

Also, @SaiSan-WMDE , I think we might be missing some component tokens for the min-height.

Reply:
Added `box-sizing: border-box;`

Reply:
Hi! Thanks for working on this, and for adding `box-sizing: border-box;`. The size of buttons looks good now. We do have min-height tokens, but we haven't been using them with the button component because they proved to be unnecessary. Should that change?

Reply:
Quick question: In `they proved to be unnecessary`, by they you mean the min-height css style or the min-height tokens?
The min-height css styles are in the figma specs and I set them as it's there. If we need to not use them, that's fine.

If that's about the tokens themselves, I'm fine with moving like this.

Reply:
Using the CSS style, and therefore the tokens to define it, seemed to not have a real impact on the component, as far as I remember. I'd say we can move on like this.

I should be rephrasing many of the Figma specs, to avoid them from being interpreted as implementation decisions. Now that UX creates component tokens, these should be considered the design source of truth by developers.

(Leaving more feedback in a separate comment)

Reply:
Done. Removed the min-height css.